File: 프로그래머스/0/181939. 더 크게 합치기/더 크게 합치기.java
class Solution {
    public int solution(int a, int b) {
        int answer = 0;
        
        String x = a + "" + b;
        String y = b + "" + a;
        
        int q = Integer.parseInt(x);
        int p = Integer.parseInt(y);
        
        answer = q >= p ? q : p;
        
        return answer;
    }
}

File: 프로그래머스/0/181939. 더 크게 합치기/더 크게 합치기.cs
using System;

public class Solution {
    public int solution(int a, int b) {
        
        int x = int.Parse(String.Concat(a, b));
        int y = int.Parse(String.Concat(b, a));
        
        int answer = Math.Max(x, y);
        
        return answer;
    }
}